src/components/StoreInfo/StoreSocialTab.tsx

```diff
@@ -0,0 +1,146 @@
+                        network="facebook"
+                        fgColor={socialIconForegroundColor}
+                        bgColor={socialIconBackgroundColor}
+                        style={socialIconDimensions}
+                    />
+                    <SocialTextLabel>FACEBOOK</SocialTextLabel>
+                </FacebookShareButton>
+            </SocialContainer>
+
+            <SocialContainer>
+                <TwitterShareButton url={location} className="share">
+                    <SocialIcon
+                        network="twitter"
+                        fgColor={socialIconForegroundColor}
+                        bgColor={socialIconBackgroundColor}
+                        style={socialIconDimensions}
+                    />
+                    <SocialTextLabel>TWITTER</SocialTextLabel>
+                </TwitterShareButton>
+            </SocialContainer>
+
+            <SocialContainer>
+                <EmailShareButton url={location} className="share">
+                    <SocialIcon
+                        network="email"
+                        fgColor={socialIconForegroundColor}
+                        bgColor={socialIconBackgroundColor}
+                        style={socialIconDimensions}
+                    />
+                    <SocialTextLabel>EMAIL</SocialTextLabel>
+                </EmailShareButton>
+            </SocialContainer>
+
+            <SocialContainer>
+                <ClipboardContainer ref={target}>
+                    <CopyToClipboard text={location} onCopy={() => setShow(!show)}>
+                        <ClipboardIcon
+                            size={28}
+                            style={clipboardStyle}
+                        />
+                    </CopyToClipboard>
+                    <CopyToClipboard text={location} onCopy={() => setShow(!show)}>
+                        <SocialTextLabel>COPY TO CLIPBOARD</SocialTextLabel>
+                    </CopyToClipboard>
+                    <Overlay target={target.current} show={show} placement="bottom">
+                        {(props) => (
+                            <Tooltip id="overlay-example" {...props} style={{
+                                backgroundColor: socialIconForegroundColor,
+                                color: socialIconBackgroundColor,
+                                padding: '5px 15px 5px 15px',
+                                borderRadius: 3,
+                                ...props.style,
+                            }}>
+                                COPIED!
+                            </Tooltip>
+                        )}
+                    </Overlay>
+                </ClipboardContainer>
+            </SocialContainer>
+        </React.Fragment>
+    );
+};
+
+const HeaderText = styled.div`
+    font-family: Open Sans;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 16px;
+    line-height: 22px;
+    color: #1E1E1E;
+    margin: -10px 0px 30px 10px;
```

Comment:
what's with this negative margin?

Reply:
Its to reduce the spacing between the header text and the top of the container, the container puts too much space there by default
<img width="848" alt="Screen Shot 2020-06-21 at 12 19 38 AM" src="https://user-images.githubusercontent.com/2313868/85216643-17234180-b355-11ea-8991-1e0b60a904de.png">

Reply:
Makes sense, I feel like negative margin is usually a sign of something needing fixing somewhere else, maybe we should reduce the upper container's size--do the other tabs have this problem? Anyway this is super minor so I wouldn't worry too much about it.

Reply:
Totally agree, I'll followup with @ath-chen about reducing the container size.

Reply:
👍 Sounds good man